test/ui-test/testSuites/suite_status/shared/steps/chatSteps.py

```diff
@@ -63,6 +63,11 @@ def step(context, message):
 def step(context, message_index, message):
     _statusChat.reply_to_message_at_index(message_index, message)
     _statusChat.verify_last_message_sent(message)
+    
+@Then("the user can edit the message at index |any| and add to the end \"|any|\"" )
+def step(context, message_index, message):
+    _statusChat.edit_message_at_index(message_index, message)
+    _statusChat.verify_last_message_sent(message)
```

Comment:
I wonder how this test passes. It adds ` me` at the end of the message, but instead of testing for `Edit me`, it tests only for ` me`.

Reply:
@jrainville cos test `verify` method use `contains` not `equal`

Reply:
Ah right. It would probably still be better to test the whole message.

My tip would be to make the `edit_message_at_index` function do a `ctrl+a` + `delete`, like I did in the edit channel/community functions.
That way the new message you pass is the full message. 
Then the validation of `message` will be valid (I'd also rename `message`  to `new_message`)

Reply:
I also suggest here to check that the edited message has the `mark` `(Edited)` at the end.